src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,17 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+        int userScope = -1;
+        for (String user : records) {
+            String[] userRecords = user.split(":");
+            if (userRecords[0].equals(email) && !email.matches(".*[0-9]")) {
```

Comment:
```suggestion
            if (userRecords[0].equals(email)) {
```
second check is redundant

Reply:
done